doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -138,13 +141,13 @@ Automatic declaration of schemes
 Combined Scheme
 ~~~~~~~~~~~~~~~~~~~~~~
 
-.. cmd:: Combined Scheme @ident from {+, @ident__i}
+.. cmd:: Combined Scheme @ident from {+, @ident }
```

Comment:
OTOH, in this case, removing the indices creates ambiguity, so if you don't like the `__i`, please introduce a `__def` for the first `@ident`.